cmd/root.go

```diff
@@ -22,6 +25,14 @@ var rootCmd = &cobra.Command{
 	Short: "eirini-loggregator-bridge streams Eirini application logs to CloudFoundry loggregator",
 	Run: func(cmd *cobra.Command, args []string) {
 		var err error
+
+		LogDebug("Namespace: ", fmt.Sprintf("%s", config.Namespace))
+		LogDebug("Loggregator-endpoint: ", fmt.Sprintf("%s", config.LoggregatorEndpoint))
+		LogDebug("Loggregator-ca-path: ", fmt.Sprintf("%s", config.LoggregatorCAPath))
+		LogDebug("Loggregator-cert-path: ", fmt.Sprintf("%s", config.LoggregatorCertPath))
+		LogDebug("Loggregator-key-path: ", fmt.Sprintf("%s", config.LoggregatorKeyPath))
```

Comment:
```suggestion
		LogDebug("Namespace: ", config.Namespace)
		LogDebug("Loggregator-endpoint: ", config.LoggregatorEndpoint)
		LogDebug("Loggregator-ca-path: ", config.LoggregatorCAPath)
		LogDebug("Loggregator-cert-path: ", config.LoggregatorCertPath)
		LogDebug("Loggregator-key-path: ", config.LoggregatorKeyPath)
```
I get a linter warning that these are already strings.